proto/panacea/datapool/v2/query.proto

```diff
@@ -35,6 +35,11 @@ service Query {
   rpc DataValidationCertificates(QueryDataValidationCertificatesRequest) returns (QueryDataValidationCertificatesResponse) {
     option (google.api.http).get = "/panacea/datapool/v2/data_validation_certificates/pool/{pool_id}/round/{round}";
   }
+
+  // DataPassRedeemReceipts returns DataPassRedeemReceipts
+  rpc DataPassRedeemReceipts(QueryDataPassRedeemReceiptsRequest) returns (QueryDataPassRedeemReceiptsResponse) {
+    option (google.api.http).get = "/panacea/datapool/v2/data_pass_redeem_receipts/pool/{pool_id}";
+  }
```

Comment:
This query returns the list of `DataPassRedeemReceipts`.
IMO, we need a query which returns a `DataPassRedeemReceipt` by token id. It may be used in [data download api](https://github.com/medibloc/panacea-data-market-validator/issues/49) of data validator to get and validate `DataPassRedeemReceipt`. 
Buyer can buy multiple `DataPass`es and redeem them, so we have to make the redeemed `DataPass` and its receipt be specific

Reply:
The query a `DataPassRedeemReceipt` was implemented in the branch #322 .